server/src/services/initialization/onInitialize.ts

```diff
@@ -40,22 +44,49 @@ export const onInitialize = (serverState: ServerState) => {
       workspaceFolders,
     });
 
-    // fetch available solidity versions
-    await updateAvailableSolcVersions(serverState);
+    // fetch available solidity versions and feature flags
+    const [flags] = await Promise.all([
```

Comment:
super nit suggestion, just for readability.

```suggestion
    const [flags, _] = await Promise.all([
```